### PR TITLE
docs/variants/protectli_vp2410: add documentation for VP2410

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

File: docs/variants/protectli_vp2410/hardware-matrix.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| **Ethernet**           | 4x intel i211 (on-board)                                 |
+| **Attached devices**   | 1. SanDisk USB 3.2Gen1 16 GB                             |
+|                        | 2. SanDisk USB 3.2Gen1 16 GB                             |
+|                        | 4. USB Type-C Hub Pro UCN3286                            |
+| **USB Keyboard**       | DELL KB216                                               |
+| **MMC drive**          | SAMSUNG 8GTF4R (on-board)                                |
+| **TPM**                | PC Engines TPM1A LPC TPM                                 |
+| **Power supply**       | Channel Well Technology 12V, 5.0A 60W                    |
+
+> Note, that in **RAM** section all used during verification procedure
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp2410/hardware-matrix.md:33]: Only two? Not only one?

File: docs/variants/protectli_vp2410/hardware-matrix.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    modules have been listed. Device has only two RAM mounting slots.
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp2410/hardware-matrix.md:34]: Not only one RAM mounting slot?

Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp2410/hardware-matrix.md:34]: Not only one RAM mounting slot?

File: docs/variants/protectli_vp2410/hardware-matrix.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+| **CPU**                | Intel Celeron J4125                                      |
+| **SSD**                | M.2 SSD SATA: Hoodisk SSL032GTTC7-S9A-2S                 |
+|                        | external SATA 2.5 inch: Goodram SSDPR-CL100-240-G2 240GB |
+| **RAM**                | CRUCIAL CT4G4SFS824A                                     |
+|                        | SAMSUNG M471A4G43MB1-CTD                                 |
+|                        | SAMSUNG M471A1K43CB1-CTD                                 |
+|                        | SAMSUNG M471A5244BB0-CRC                                 |
+|                        | KINGSTON KVR24S17S8/8                                    |
+|                        | KINGSTON KVR26S19S8/16                                   |
+| **Wireless card**      | Intel Wi-Fi 6 AX201                                      |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp2410/hardware-matrix.md:21]: Currently, we have mounted: `Intel Wi-Fi 6 AX200 (rev 1a)`

Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp2410/hardware-matrix.md:21]: Currently, we have mounted: `Intel Wi-Fi 6 AX200 (rev 1a)`

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/protectli_vp2410/hardware-matrix.md (hunk 2 above)
Comment (miczyg1) [reply docs/variants/protectli_vp2410/hardware-matrix.md]: Yes, fixed

File: docs/variants/protectli_vp2410/hardware-matrix.md (hunk 3 above)
Comment (miczyg1) [reply docs/variants/protectli_vp2410/hardware-matrix.md]: Fixed

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Comment by sulewskiprzemyslaw** [comment]
@miczyg1 @macpijan I don't have any further objections

File: docs/variants/protectli_vp2410/installation-manual.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Protectli Dasharo on VP2410 - installation manual
+
+Flashing coreboot can be done from Linux using flashrom with the internal
```
Comment (macpijan) [inline docs/variants/protectli_vp2410/installation-manual.md:3]:
```suggestion
Flashing Dasharo firmware can be done from Linux using flashrom with the internal
```

**Review by macpijan — APPROVED** [review]

**Pull request merged**